C++/include/data_structures/doubly_linked_list.cpp

```diff
@@ -0,0 +1,301 @@
+/*
+ * C++ Program to Implement Doubly Linked List 
+ */
+#include<iostream>
+#include<cstdio>
+#include<cstdlib>
+/*
+ * Node Declaration
+ */
+using namespace std;
+struct node
+{
+    int info;
+    struct node *next;
+    struct node *prev;
+}*start;
```

Comment:
`*start` should be a private class variable that is initialized in its constructor.